labs/aries-fabric.md

```diff
@@ -0,0 +1,29 @@
+# Aries Wrapper For Fabric
+
+# Description
+
+Self - Sovereign Identity application is an application by which you are able to keep your identity secured and at the same time also prove your identity to the verifier without revealing them. These applications are built using Hyperledger Aries, which depends on Indy as a Distributed Ledger and Ursa as a cryptography library. Currently, the most used Blockchain platform is Hyperledger Fabric. And the businesses that are building solutions using Fabric want to have some sort of identity verification to be done in their platform. This is the reason, why this project has come into existence. Once done it would enable Aries to interact with Fabric as a ledger, which would then enable SSI applications to be built on top of Fabric.
+
+# Scope of Lab
+
+As Hyperledger's mission is to increase the adoption of Blockchain technologies for business, this project would enable business that are using Fabric for building their blockchain solutions to have identity verification process to be in-built in their system. The future developements that it would undergo once it is set as a project in lab are :
```

Comment:
```suggestion
As Hyperledger's mission is to increase the adoption of Blockchain technologies for business, this will enable businesses that are using Fabric for building their blockchain solutions to have identity verification process to be built-in into their system. The future developments that it will involve once it is set as a lab are :
```

Reply:
I have updated the proposal with the mentioned changes.